app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/NiaAppState.kt

```diff
@@ -117,6 +120,20 @@ class NiaAppState(
             initialValue = false,
         )
 
+    private val errorMessages = MutableStateFlow<List<ErrorMessage>>(emptyList())
+    val errorMessage: StateFlow<String?> = errorMessages.map { it.firstOrNull()?.message }.stateIn(
```

Comment:
Why use nullable `String?` and init null, instead empty string?

Reply:
In this case, the null value indicates an empty errormessages list, whereas an empty string would indicated an error in the list with no message. 

But after considering this, I have made an update to check the error messages by id, so please check that as well. Thank you!